conftest.py

```diff
@@ -1,24 +1,20 @@
-import shutil
-
 import numpy as np
 import pytest
 from requests import RequestException
 
 from pyquil.api import (
     QVMConnection,
     QVMCompiler,
-    ForestConnection,
-    get_benchmarker,
-    local_forest_runtime,
+    Client, BenchmarkConnection,
```

Comment:
Looks like this needs a `black` run

Reply:
I've noticed these. Strangely, black doesn't catch them... but I'll do it manually